### ISSUE TITLE
Commands hang with no output

### ISSUE THREAD

**Issue by bifkit** [docker/cli#1966]
**Description**
After installing Docker EE via the [official directions](https://hub.docker.com/editions/enterprise/docker-ee-server-windows) on a shiny new server running Windows Server 2019 (1809), I cannot successfully execute certain commands. The process begins, but then hangs silently, producing no output, logs, or any other useful information.

**Steps to reproduce the issue:**
1. Install Docker EE via PowerShell 5.1.
2. Try to start a container, eg. `docker container run hello-world:nanoserver`
3. Feel another part of your soul wither away as the command prompt silently taunts you, never exiting.

**Describe the results you received:**
Initially, some commands execute without any problem: pull; version; info; images; ps; etc.. But once the run command hangs, no other commands will complete until I restart the Docker service, or completely reboot. Adding the -it or -D flags does not change the result.

**Describe the results you expected:**
"Hello world!"

**Additional information you deem important (e.g. issue happens only occasionally):**
I have uninstalled and reinstalled several times trying to troubleshoot, but I get the same result. When I tried to remove the docker directory from %programdata%, I had some trouble: the permissions on that directory and its children are a mess. I couldn't even view the owner of the directory. I had to take ownership and push new permissions to all the subdirectories in order to delete everything.
I don't know if that's even related to my issue, but it's pretty much the only thing that I can understand at this point.

**Output of `docker version`:**

```
Client: Docker Engine - Enterprise
 Version:           18.09.6
 API version:       1.39
 Go version:        go1.10.8
 Git commit:        1578dcadd2
 Built:             05/04/2019 02:34:11
 OS/Arch:           windows/amd64
 Experimental:      false

Server: Docker Engine - Enterprise
 Engine:
  Version:          18.09.6
  API version:      1.39 (minimum version 1.24)
  Go version:       go1.10.8
  Git commit:       1578dcadd2
  Built:            05/04/2019 02:32:24
  OS/Arch:          windows/amd64
  Experimental:     false
```

**Output of `docker info`:**

```
Containers: 1
 Running: 0
 Paused: 0
 Stopped: 1
Images: 1
Server Version: 18.09.6
Storage Driver: windowsfilter
 Windows:
Logging Driver: json-file
Plugins:
 Volume: local
 Network: ics l2bridge l2tunnel nat null overlay transparent
 Log: awslogs etwlogs fluentd gelf json-file local logentries splunk syslog
Swarm: inactive
Default Isolation: process
Kernel Version: 10.0 17763 (17763.1.amd64fre.rs5_release.180914-1434)
Operating System: Windows Server 2019 Standard Version 1809 (OS Build 17763.557)
OSType: windows
Architecture: x86_64
CPUs: 32
Total Memory: 31.84GiB
Name: UoC-Server
ID: EH3U:ZPTM:VT4D:XZRJ:EN3I:2SLF:W5C3:J6HO:7X55:GS2Z:BJX2:OEVW
Docker Root Dir: C:\ProgramData\docker
Debug Mode (client): false
Debug Mode (server): false
Registry: https://index.docker.io/v1/
Labels:
Experimental: false
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**
Windows Server 2019, physical installation.

**Comment by ddebroy** [comment]
@bifkit Do the events log (System and Application logs specifically) show anything interesting around Docker when you hit this condition?

**Comment by bifkit** [comment]
@ddebroy I haven't gone through those logs yet, but I do have an update.

During my uninstall/reinstall frenzy, I noticed that the uninstall process reported that a network was removed. I definitely didn't create any networks via docker, but the name of the network seemed familiar.
Apparently, I had created a virtual switch in the Hyper-V management console, but I had never configured anything. After deleting that switch, Docker now runs as expected.

I will defer to the devs on whether this ticket ought to be closed or further explored for why a Hyper-V network issue seems to put everything in suspended animation.
